fieldtypes/text/text.vue

```diff
@@ -42,6 +60,21 @@
             return {
                 fieldid: _.uniqueId('formmakerfield_')
             };
+        },
+
+        created: function () {
+            //defaults admin
+            this.field.data.minLength = this.field.data.minLength || 0;
+            this.field.data.maxLength = this.field.data.maxLength || 0;
+        },
+
+        computed: {
+            minLength: function () {
+                return this.field.data.minLength && !this.isAdmin ? this.field.data.minLength : false;
```

Comment:
This should also be applied in admin. This is for the default value, that should be restricted as well. Required is always off in admin because a default value is not required.